src/routes/exercise-finished/__tests__/ExerciseFinishedScreen.spec.tsx

```diff
@@ -54,7 +55,15 @@ describe('ExerciseFinishedScreen', () => {
       closeExerciseAction: route.params.closeExerciseAction,
     })
   })
-
+  it('should render repetition modal if a module is finished', () => {
```

Comment:
Thanks for adding a test :+1: 

Please add a blank line before and after the test, also it would be nice to check the modal is not rendered where it should not be displayed.

Reply:
@SteffiStuffel I added several test cases to check it